dhis-2/dhis-api/src/main/java/org/hisp/dhis/dataelement/DataElementOperand.java

```diff
@@ -385,6 +399,11 @@ public void setAttributeOptionCombo( CategoryOptionCombo attributeOptionCombo )
     // hashCode, equals and toString
     // -------------------------------------------------------------------------
 
+    public CategoryOption getCategoryOption()
+    {
+        return this.categoryOption;
+    }
```

Comment:
This should be in the section starting with the comment "Getters & setters", not in the section "hashCode, equals and toString".

Reply:
done